src/utils/lsk.js

```diff
@@ -0,0 +1,11 @@
+import BigNumber from 'bignumber.js';
+
+BigNumber.config({ ERRORS: false });
+
+export const fromRawLsk = value => (
```

Comment:
Now that you've implemented these utils, would be neat if you can use them in `liskAmount` component. Just a suggestion.